packages/rps/src/redux/open-games/saga.ts

```diff
@@ -119,7 +119,7 @@ const openGameTransformer = dict => {
 function* openGameSyncer() {
   yield fork(
     reduxSagaFirebase.database.sync,
-    'challenges',
+    `${FIREBASE_PREFIX}/challenges`,
```

Comment:
Since it seemed easy to miss some of these, does it make sense to centralize the logic that constructs these paths?